content/contact.md

```diff
@@ -1,12 +1,11 @@
 ---
-date: 2016-12-30T21:30:24-08:00
-title: contactPageTitle
+title: Out contacts
+description: Our HQ is in Tallin, Estonia. Development and scientific team is located in Minsk, Belarus. We respond to every email.
```

Comment:
... Most of our scientists, engineers, and industry experts are working hard in Minsk, Belarus.